### PR TITLE
Refactor for git-lite cli

### PR DESCRIPTION
**Pull request by bwalsh**
This PR:
* updates documentation for the "git lite" CLI

This PR is one of a set: 
  * https://github.com/https://github.com/ACED-IDP/gen3_util/pull/42
  * https://github.com/ACED-IDP/aced_etl_pod/pull/21
  * https://github.com/https://github.com/ACED-IDP/aced-idp.github.io/pull/11 
  * https://github.com/ACED-IDP/submission/pull/16

### CODE REVIEW

**Comment by bwalsh** [comment]
@matthewpeterkort  can you create an issue or expand on your comment in the your review here?

https://github.com/ACED-IDP/gen3_util/pull/42#issuecomment-1900967019

**Comment by lbeckman314** [comment]
Preview site for this PR is available at https://deploy-preview-13--luminous-boba-a6e270.netlify.app/

**Comment by lbeckman314** [comment]
LGTM! Very minor UX question — do we prefer the navigation menu to be expanded on not on page load?

## Expanded Nav

Screenshot 2024-01-22 at 11 23 04 AM

## Collapsed Nav

Screenshot 2024-01-22 at 11 22 30 AM

File: docs/workflows/restrict-access.md [hunk 1]
```diff
@@ -11,22 +11,22 @@ For example, given the example above, let's say we needed to restrict access to
...
-gen3_util projects new --project_id=test-myproject_restricted
+g3t init --project_id=test-myproject
+g3t init --project_id=test-myproject_restricted
 ```
 
 Next, lets add a user to the primary project, but **not** the restricted project:
 
 ```sh
-gen3_util users add --username someone@example.com --project_id=test-myproject
+g3t untilities users add --username someone@example.com --project_id=test-myproject
```
Comment (lbeckman314) [inline docs/workflows/restrict-access.md:21]: Possible small typo:

```diff
- g3t untilities ...
+ g3t utilities ...
```

**Review by lbeckman314 — APPROVED** [review]
LGTM! Very comprehensive and well structured. UX question (nav expansion) and possible small typo noted, otherwise all approved.

**Comment by matthewpeterkort** [comment]
> @matthewpeterkort can you create an issue or expand on your comment in the your review here?
> 
> [ACED-IDP/gen3_util#42 (comment)](https://github.com/ACED-IDP/gen3_util/pull/42#issuecomment-1900967019)

Ok. https://github.com/ACED-IDP/aced-idp.github.io/issues/14

File: docs/workflows/restrict-access.md (hunk 1 above)
Comment (bwalsh) [reply docs/workflows/restrict-access.md]: thanks.  fixed it

File: docs/workflows/metadata.md [hunk 2]
```diff
@@ -1,81 +1,58 @@
...
+* The metadata files can be created at any time, but the system will validate them before the changes are committed.
 
-Convert the tabular data to FHIR.
-This is a local operation that does not require access to the portal. p.
+e.g. The `g3t utilities meta create` will use the metadata identifiers specified on the `g3t add` command:
 
 ```sh
-gen3_util meta from_tabular  /tmp/$PROJECT_ID  /tmp/$PROJECT_ID/tabular
-
+g3t add myfile.cram  --patient P0 --specimen P0-BoneMarrow --task_id P0-Sequencing
```
Comment (matthewpeterkort) [inline docs/workflows/metadata.md:39]: --task_id typo should be --task

**Pull request merged**